README.md

```diff
@@ -1,52 +1,109 @@
+## Setup and Deploy
+The first step to getting setup is to [sign up][signup]. 
 
-### Dependencies
+Update the .env file with the following:
+1. Your App Id that you can get from [Platform](https://platform.symbl.ai)
+2. Your App Secret that you can get from [Platform](https://platform.symbl.ai)
 
-```package.json
-"dependencies": {
-	"react": "16.8.6",
-	"react-dom": "16.8.6",
-	"redux": "3.7.2",
-	"react-redux": "7.1.0",
-	"jquery": "^3.4.1",
-	"lodash": "^4.17.14",
-	"redux-thunk": "2.2.0"
-}
+Run the follwing npm commands:
+1. `npm install` to download all the node modules
+2. `node app.js` to start the node server
+
+Navigate to localhost:5000 to view the app
+1. Enter the email address you would like the meeting summary sent to
+2. Enter a Meeting Name Identifier
+3. Paste the full meeting invite for the Zoom meeting you wish to connect to. Ex.
+    * ```
+        Symbl is inviting you to a scheduled Zoom meeting.
+
+        Topic: Symbl Personal Meeting Room
+
+        Join Zoom Meeting
+        https://us02web.zoom.us/j/55555555?pwd=<meeting-password>
+
+        Meeting ID: 555 5555 555
+        Passcode: 55555
+        One tap mobile
+        +16699009128,,2323522600# US (San Jose)
+        +12532158782,,2323522600# US (Tacoma)
+
+        Dial by your location
+                +1 669 900 9128 US (San Jose)
+                +1 253 215 8782 US (Tacoma)
+                +1 346 248 7799 US (Houston)
+                +1 646 558 8656 US (New York)
+                +1 301 715 8592 US (Washington D.C)
+                +1 312 626 6799 US (Chicago)
+        Meeting ID: 232 352 2600
+        Find your local number: https://us02web.zoom.us/u/kz2YbGRTL
+
+        Join by SIP
+        5555555555@zoomcrc.com
+
+        Join by H.323
+        162.255.37.11 (US West)
+        162.255.36.11 (US East)
+
+        Passcode: 555555
+    ```
+4.  Submit and Symbl will join via Telephony API dial in from PSTN 12015947998
+
+## Dependencies
+
+```json
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "lodash": "^4.17.15",
+    "path": "^0.12.7",
+    "request": "^2.88.2",
+    "request-promise": "^4.2.5",
+    "symbl-node": "^1.0.3",
+    "url": "^0.11.0"
+  }
 ```
 
-## References
-
-Feel free to fork any of the projects here to use on your own and if you have any code improvements, make a pull request and the request will be reviewed by one of our admins.
-
-For a sample reference implentation using Symbl, take a look at our [Platform](https://platform.symbl.ai).
+## Community
 
-If you have questions, bugs to report or feature suggestions, join our [Dev Community](https://community.symbl.ai/).
+If you liked our integration guide, please star our repo! If you have any questions, feel free to reach out to us at devrelations@symbl.ai or through our Community Slack at https://developer.symbl.ai/community/slack or our [developer community][developer_community]. 
```

Comment:
@HuckMatt Please reformat. See below. 

<img width="1156" alt="Screen Shot 2021-01-22 at 3 58 34 PM" src="https://user-images.githubusercontent.com/77749193/105561089-b08b4b00-5cca-11eb-81ba-4b322abc5115.png">

Reply:
@eric-giannini reformatted in latest commit